### ISSUE TITLE
Cannot reopen Complete project

### ISSUE THREAD

**Issue by acshields** [appirio-tech/connect-app#4262]
This project is in Complete status and needs to be reopened, but the option is not currently available: https://connect.topcoder.com/projects/658

The issue was originally reported to support in ticket #76573.

Goal: Identify root cause and solve.

**Comment by maxceem** [comment]
@acshields at the moment no-one can change project status once it's completed. Should Topcoder Managers, Copilots and Admins be able to change status when it's completed?

**Comment by acshields** [comment]
@maxceem - Yes, I think that would be good.

**Comment by acshields** [comment]
@maxceem - I don't think work on this has started, right? Is this one where the necessary changes can be made through the community, without direct effort needed from you or Rishi?

**Comment by maxceem** [comment]
@acshields yes, this could be done by a community member by moderate control from the copilot.

**Comment by acshields** [comment]
@maxceem - Alright, thanks for your input. I will leave this one prioritized for now and if we can fit this fix in sometime during the next 6 weeks, awesome.

**Comment by acshields** [comment]
### Updated Requirements
Copilots, Admins, and Topcoder Managers should be able to change a project status from "Complete" to another status, if necessary.

**Comment by vikasrohit** [comment]
I know we are not targeting it during our shape up project, however, just like to raise the point that we should be looking at consequences of allowing changing project status from `Complete` or `Cancelled`. Specifically, how the things would flow to the SFDC.

**Comment by maxceem** [comment]
> I know we are not targeting it during our shape up project, however, just like to raise the point that we should be looking at consequences of allowing changing project status from Complete or Cancelled. Specifically, how the things would flow to the SFDC.

@vikasrohit we can include this task into work for the next 2 weeks if you didn't find any drawbacks from SFDC point of view.

**Comment by acshields** [comment]
@maxceem - Right now, I don't think that SFDC is consuming the "Complete" status change for any purpose. @vikasrohit do you know?

I do think they are consuming the "Cancelled" status change (i.e. a project is moved to Cancelled) to disqualify a lead. However, I don't think that they have anything that listens for the change from Cancelled to a different status.

I am guessing if anything, I will just need to consult with them to give them ideas on other automation/changes that they could get in place in SFDC using these status changes. But would be interested to hear from Vikas.

**Comment by vikasrohit** [comment]
@acshields I don't think any one is consuming the `complete` status event, but better to check with SFDC team.

**Comment by maxceem** [comment]
As I understand we don't include this into work until confirmed with SFDC team.

**Comment by vikasrohit** [comment]
@acshields moving it out of current bug cycle.
